Formula/alpine.rb

```diff
@@ -31,6 +29,9 @@ class Alpine < Formula
     depends_on "linux-pam"
   end
 
+  # patch for macOS obtained from developer
+  patch :DATA
```

Comment:
Any upstream commit or issue reference for this?

Reply:
I'll add one as soon as the developer commits these to the repo.